lib/model/CEventRateModel.cc

```diff
@@ -478,9 +478,7 @@ void CEventRateModel::debugMemoryUsage(core::CMemoryUsage::TMemoryUsagePtr mem)
 }
 
 std::size_t CEventRateModel::memoryUsage() const {
-    return this->CIndividualModel::memoryUsage() +
-           core::CMemory::dynamicSize(m_InterimBucketCorrector);
```

Comment:
Where is the memory for the interim bucket corrected accounted for now?

Reply:
It is getting accounted for in `this->CIndividualModel::computeMemoryUsage()`. This is all tied up with the model memory estimation process we use, i.e. measuring the computed memory usage periodically then using a regression on those measurements. The extra memory used is effectively accounted for in that regressions' parameters.